libnmstate/schemas/operational-state.yaml

```diff
@@ -241,6 +254,23 @@ definitions:
                       type: integer
                     max-age:
                       type: integer
+                vlan-filtering:
+                  type: boolean
+                vlan-default-pvid:
```

Comment:
The simplest course of action is to not expose this setting in the model, and set it to 0 (which indicates NetworkManager to **not** tag untagged frames) in every created BridgeSettings object.

It can always be set via the boolean 'pvid'.

It would simplify the implementation effort quite a lot.